tools/buildsys/src/args.rs

```diff
@@ -165,17 +198,23 @@ fn sensitive_env_vars(build_type: BuildType) -> impl Iterator<Item = &'static st
 }
 
 /// Emits the cargo directives for a the list of sensitive environment variables for a given
-/// `[BuildType]`.
+/// `[BuildType]`, unless the `[BuildType]` is Repack.
 pub(crate) fn rerun_for_envs(build_type: BuildType) {
-    for var in sensitive_env_vars(build_type) {
-        println!("cargo:rerun-if-env-changed={}", var)
+    match build_type {
+        BuildType::Repack => (),
+        _ => {
+            for var in sensitive_env_vars(build_type) {
+                println!("cargo:rerun-if-env-changed={}", var)
+            }
+        }
```

Comment:
That's what I would have thought, but without it I got a wall of output.

```
cargo:rerun-if-env-changed=BUILDSYS_ARCH
cargo:rerun-if-env-changed=BUILDSYS_NAME
cargo:rerun-if-env-changed=BUILDSYS_OUTPUT_DIR
cargo:rerun-if-env-changed=BUILDSYS_PACKAGES_DIR
cargo:rerun-if-env-changed=BUILDSYS_PRETTY_NAME
cargo:rerun-if-env-changed=BUILDSYS_ROOT_DIR
cargo:rerun-if-env-changed=BUILDSYS_STATE_DIR
cargo:rerun-if-env-changed=BUILDSYS_TIMESTAMP
cargo:rerun-if-env-changed=BUILDSYS_VARIANT
cargo:rerun-if-env-changed=BUILDSYS_VERSION_BUILD
cargo:rerun-if-env-changed=BUILDSYS_VERSION_IMAGE
cargo:rerun-if-env-changed=TLPRIVATE_SDK_IMAGE
```

Reply:
> That's what I would have thought, but without it I got a wall of output.

That's unexpected - it'd be worth root-causing that rather than sidestepping the machinery here, which feels like a bit of a hack.